src/components/GameItem.js

```diff
@@ -16,14 +28,26 @@ class GameItem extends Component {
     };
 
     // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+    const icon = ItemIcons[this.props.type];
+
+    // Change css if spotted Item is litter
+    let spottedItemStyle  = " " 
+    if (this.state.spottedItem ) { 
+      this.props.type === "litter" ? spottedItemStyle = " spotted-litter" : spottedItemStyle = " spotted-nature";
+    }
 
     return (
-      <div className="game-item" style={itemStyle}>
-        <img src={icon} alt="Item" className="icon-item"></img>
+      <div className={`game-item${spottedItemStyle}`} style={itemStyle}>
```

Comment:
Instead of assuming that `spottedItemStyle`'s value will have a space in the beginning, you can rely on string interpolation to put in that space, so this can look like `... className={\`game-item ${spottedItemStyle}\`}`, and let the values of `spottedItemStyle` be `spotted-litter` or `spotted-nature` without the space :)